src/test/features/dashboard/routes/claimStatus/full-admission.ts

```diff
@@ -0,0 +1,714 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from 'test/routes/hooks'
+import 'test/routes/expectations'
+
+import { Paths } from 'dashboard/paths'
+
+import { app } from 'main/app'
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import * as moment from 'moment'
+import { MomentFactory } from 'shared/momentFactory'
+import { NumberFormatter } from 'utils/numberFormatter'
+
+import {
+  baseFullAdmissionData,
+  basePayByInstalmentsData,
+  basePayBySetDateData,
+  basePayImmediatelyData,
+  baseResponseData
+} from 'test/data/entity/responseData'
+
+import {
+  claimantResponseAt,
+  claimantReferredToJudgeResponse,
+  claimantReferredToJudgeResponseForInstalments,
+  claimantAcceptRepaymentPlan,
+  settlementOfferBySetDate,
+  settlementOfferByInstalments,
+  settlementOfferAcceptBySetDate,
+  settlementOfferAcceptInInstalment,
+  settledWithAgreementBySetDate,
+  settledWithAgreementInInstalments,
+  settledWithAgreementBySetDatePastPaymentDeadline,
+  settledWithAgreementInInstalmentsPastPaymentDeadline,
+  defendantRejectedSettlementOfferAcceptBySetDate,
+  defendantRejectedSettlementOfferAcceptInInstalments,
+  claimantAcceptRepaymentPlanByDetermination,
+  claimantAcceptRepaymentPlanInInstalmentsByDetermination,
+  setByDatePaymentIntentionPastDeadline,
+  defendantOffersSettlementByInstalments
+
+} from 'test/data/entity/fullAdmission'
+
+const cookieName: string = config.get<string>('session.cookieName')
+
+const fullAdmissionClaim = {
+  ...claimStoreServiceMock.sampleClaimObj,
+  responseDeadline: MomentFactory.currentDate().add(5, 'days'),
+  response: {
+    ...baseResponseData,
+    ...baseFullAdmissionData
+  },
+  ...claimantResponseAt
+}
+
+const testData = [
+  {
+    status: 'Full admission - defendant responded to pay immediately',
```

Comment:
this should read "- defendant responded pay immediately" the " to " is implying that the responded to a pay immediately rather than responding with a pay immediately. This will need to be changed for the other tests as well. Sorry!

Reply:
done